src/main/java/com/ceos20/spring_instagram/service/CommentService.java

```diff
@@ -0,0 +1,77 @@
+package com.ceos20.spring_instagram.service;
+
+import com.ceos20.spring_instagram.domain.Comment;
+import com.ceos20.spring_instagram.domain.Post;
+import com.ceos20.spring_instagram.domain.Member;
+import com.ceos20.spring_instagram.repository.CommentRepository;
+import com.ceos20.spring_instagram.repository.PostRepository;
+import com.ceos20.spring_instagram.repository.MemberRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class CommentService {
+
+    private final CommentRepository commentRepository;
+    private final PostRepository postRepository;
+    private final MemberRepository memberRepository;
+
+    @Autowired
+    public CommentService(CommentRepository commentRepository, PostRepository postRepository, MemberRepository memberRepository) {
+        this.commentRepository = commentRepository;
+        this.postRepository = postRepository;
+        this.memberRepository = memberRepository;
+    }
+
+    // 댓글 추가
+    public Comment addComment(Long postId, Long memberId, String text) {
+        Optional<Post> post = postRepository.findById(postId);
+        Optional<Member> member = memberRepository.findById(memberId);
+
+        if (post.isEmpty() || member.isEmpty()) {
+            throw new IllegalArgumentException("잘못된 포스트 또는 회원 ID입니다.");
+        }
```

Comment:
이렇게 예외처리를 하면 Post가 빈 값인지 Member가 빈 값인지 확인해야겠죠. 조금 더 명확하게 예외 처리를 하면 좋을 것 같아요.
```suggestion
        Post post = postRepository.findById(postId)
                .orElseThrow(() -> new IllegalArgumentException("잘못된 포스트 ID입니다."));
        Member member = memberRepository.findById(memberId)
                .orElseThrow(() -> new IllegalArgumentException("잘못된 회원 ID입니다."));
```